ddtrace/contrib/aiohttp/middlewares.py

```diff
@@ -9,6 +9,9 @@
 REQUEST_CONTEXT_KEY = 'datadog_context'
 REQUEST_SPAN_KEY = '__datadog_request_span'
 
+PARENT_TRACE_HEADER_ID = 'x-datadog-trace-id'
+PARENT_SPAN_HEADER_ID = 'x-datadog-parent-id'
```

Comment:
@thehesiod I've changed the headers so that they're compliant with other integrations (like our Go client)